frontend/packages/dev-console/integration-tests/features/topology/topology_pipeline_feature-ODC-3925

```diff
@@ -0,0 +1,29 @@
+Feature: Pipeline features in topology
```

Comment:
Can you please update the feature with some more information?

Reply:
It's regarding pipelines in topology

Reply:
It should be more descriptive or please add this to the existing feature files